src/middlewared/middlewared/plugins/account.py

```diff
@@ -692,6 +685,22 @@ async def set_root_password(self, app, password, options):
         root = await self.middleware.call('user.query', [('username', '=', 'root')], {'get': True})
         await self.middleware.call('user.update', root['id'], {'password': password})
 
+    @job(lock="home_copy")
```

Comment:
We can use lock that includes `home_new` in its name so multiple independent calls could be performed simultaneously

Reply:
Good idea

Reply:
Should probably mark it as private as well

Also we should probably keep the change as minimal as possible in the 12.0 backport